hw/ip/adc_ctrl/README.md

```diff
@@ -1,6 +1,4 @@
----
-title: "Analog to Digital Converter Control Interface"
----
+# Analog to Digital Converter Control Interface
 
 # Overview
```

Comment:
I had made a commit somewhere that removes all the `# Overview`
This stops many of the section heading of having the same importance as the title.